File: FrontEnd/src/models/EMSDTO.ts
export interface EMSDTO {
  id: string;
  EMSName?: string | null;
  isActive: boolean;
  createdDate?: Date | null;
  idExam: string;
  idExamSubject: string;
  idTestSchedule: string;
  numberOfStudents?: number | null;
  numberOfLecturers?: number | null;
  idStudyGroup?: string;
  isDeleted: boolean;
}

File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/EMSDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DPD.HumanResources.Dtos.Dto
{
    public class EMSDto
    {
        public Guid Id { get; set; }
        public string? EMSName { get; set; }
        public bool IsActive { get; set; }
        public DateTime? CreatedDate { get; set; }
        public Guid IdExam { get; set; }
        public Guid IdExamSubject { get; set; }
        public Guid IdTestSchedule { get; set; }
        public int? NumberOfStudents { get; set; }
        public int? NumberOfLecturers { get; set; }
        public Guid? IdStudyGroup { get; set; }
        public bool IsDeleted { get; set; }
    }
}
